src/pages/DogEdit.js

```diff
@@ -1,9 +1,76 @@
-import React from 'react';
+import React, { useState } from "react";
+import { Form, FormGroup, Label, Input, Button } from "reactstrap";
+import { useNavigate, useParams } from "react-router-dom";
+
+const DogEdit = ({ dogs, updateDog }) => {
+  const { id } = useParams();
+  const navigate = useNavigate();
+
+  const currentDog = dogs.find((dog) => dog.id === +id);
+
+  const [editDog, setEditDog] = useState({
+    name: currentDog.name,
+    age: currentDog.age,
+    enjoys: currentDog.enjoys,
+    image: currentDog.image,
+  });
+
+
+  const handleChange = (e) => {
+    setEditDog({ ...editDog, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = () => {
+    updateDog(editDog, currentDog.id);
```

Comment:
You are passing two arguments here, however, your function in App.js only has one parameter.